app/lib/hpitt.rb

```diff
@@ -87,6 +93,40 @@ def build_trainee(csv_row)
         column_mapper[key].call(value)
       end
 
+      if trainee.international_address.present?
```

Comment:
Feels like we could break some of this up into their own methods to keep `build_trainee` from getting bigger than it already is but this isn't a blocker at all for this PR